src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) throws IOException {
```

Comment:
You should use try-with-resources or at least a try-catch block here to ensure that any resources used in writing to the file are properly closed in case of an exception.